### PR TITLE
Symintify view_as_complex and view_as_real

### PR DESCRIPTION
**Pull request by anjali411**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #89052

### CODE REVIEW

File: test/test_proxy_tensor.py [hunk 1]
```diff
@@ -1325,7 +1325,6 @@ def f(a, b, c, d, e):
     xfail('trapz', ''),  # aten.size.default - couldn't find symbolic meta function/decomposition
     xfail('trapezoid', ''),  # aten.size.default - couldn't find symbolic meta function/decomposition
     xfail('triangular_solve', ''),  # aten.triangular_solve.default - couldn't find symbolic meta function/decomposition
-    xfail('view_as_complex', ''),  # aten.view_as_complex.default - couldn't find symbolic meta function/decomposition
```
Comment (ezyang) [inline test/test_proxy_tensor.py:1328]: hmm, how come there is no `view_as_real` here...

**Review by ezyang — APPROVED** [review]

File: test/test_proxy_tensor.py (hunk 1 above)
Comment (anjali411) [reply test/test_proxy_tensor.py:1328]: yeah the allowed tests is set to torch.float so any op that doesn't currently support floating point type tensors is not currently tested by proxy_tensor tests

**Pull request closed without merging**